ide-common/src/main/kotlin/org/digma/intellij/plugin/posthog/ContainerEngineStartupActivity.kt

```diff
@@ -0,0 +1,55 @@
+package org.digma.intellij.plugin.posthog
+
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.startup.StartupActivity
+import org.digma.intellij.plugin.log.Log
+import java.util.concurrent.TimeUnit
+
+class ContainerEngineStartupActivity : StartupActivity {
+    private val timeoutInSec = 5L
+    private val logger = Logger.getInstance(ContainerEngineStartupActivity::class.java)
+    private val isWindows: Boolean = System.getProperty("os.name").startsWith("windows", true)
+
+    enum class ContainerEngine {
+        DOCKER,
+        PODMAN
+    }
+
+    override fun runActivity(project: Project) {
+        val containerPlatform = getInstalled(project)?.toString()
+        ActivityMonitor.getInstance(project).registerContainerEngine(containerPlatform)
+    }
+
+    private fun getInstalled(project: Project): ContainerEngine? {
```

Comment:
please void the null.
```suggestion
    private fun getInstalled(project: Project): ContainerEngine {
```

Reply:
removed "?"